common/lib/xmodule/xmodule/video_module/video_module.py

```diff
@@ -427,6 +434,35 @@ def __init__(self, *args, **kwargs):
         if not self.fields['download_track'].is_set_on(self) and self.track:
             self.download_track = True
 
+    def validate(self):
+        """
+        Validates the state of this video Module Instance. This
+        is the override of the general XBlock method, and it will also ask
+        its superclass to validate.
+        """
+        validation = super(VideoDescriptor, self).validate()
+        if not isinstance(validation, StudioValidation):
+            validation = StudioValidation.copy(validation)
```

Comment:
I think it's the way to have validation copy object when we don't have one. Already being used in couple of places.